pkg/configuration/context.go

```diff
@@ -1,44 +1,39 @@
+// See parameters.go for this package's comment
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [package comment should be of the form "Package configuration ..."](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)